contracts/sol6/KyberStorage.sol

```diff
@@ -24,6 +25,7 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers {
     IKyberNetworkProxy[] internal kyberProxyArray;
 
     mapping(bytes32 => address[]) public reserveIdToAddresses;
+    mapping(bytes32 => address) public reserveRebateWallet;
     mapping(address => bytes32) internal reserveAddressToId;
```

Comment:
Hmmm, should we have this public as well? Or keep reserveIdToAddresses internal? For consistency. Seems weird to me that one side is public, the other side internal.

Reply:
so if its internal
We query using getReserveData?

Reply:
Updated